packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesConfiguration.kt

```diff
@@ -115,6 +115,25 @@ class UpdatesConfiguration private constructor (
 
     private const val UPDATES_CONFIGURATION_RELEASE_CHANNEL_DEFAULT_VALUE = "default"
     private const val UPDATES_CONFIGURATION_LAUNCH_WAIT_MS_DEFAULT_VALUE = 0
+
+    private fun getRuntimeVersion(context: Context?): String? {
```

Comment:
Sadly that does not work when reading from the expo-updates library. It seems to try to find the string in its own `src/main/res` instead of the app's resources.

Reply:
Weird, it seems to be working for me.

How I repro'd:
1. Check out this PR
2. Change this code to just do `context.getMetadataValue<Any>("expo.modules.updates.EXPO_RUNTIME_VERSION")?.toString()`
3. `npx create-expo-app wat`
4. `npx expo install expo-updates`
5. change the dependency in package.json to be the local path to the expo-updates package within the expo/expo repo
6. `eas update:configure`
7. `npx expo prebuild`
8. Change the AndroidManifest value of the runtime version thing to `@strings/hellowat`
9. Add `<string name="hellowat">1.0.0</string>` to application strings.xml
10. Enable debugging in project.ext.react (so that bundle builds)
11. Add breakpoint in UpdatesConfiguration.kt
12. See it resolves to `1.0.0`.

Reply:
@wschurman I've changed the config plugin to generate the metadata correctly with the android:value attribute, and reverted the unnecessary changes to expo-updates Android native code.